### PR TITLE
Add User tests, better Validation and small fixes

### PR DESCRIPTION
**Pull request by skamoen**

### CODE REVIEW

File: src/main/java/ch/wisv/areafiftylan/dto/ProfileDTO.java [hunk 1]
```diff
@@ -8,16 +8,16 @@
 public class ProfileDTO {
```
Comment (TimvdLippe) [inline src/main/java/ch/wisv/areafiftylan/dto/ProfileDTO.java]: Lombok would be sooo nice for this class. No more manual generated setters.

File: src/test/java/ch/wisv/areafiftylan/UserRestIntegrationTest.java [hunk 2]
```diff
@@ -1,17 +1,91 @@
 package ch.wisv.areafiftylan;
```
Comment (TimvdLippe) [inline src/test/java/ch/wisv/areafiftylan/UserRestIntegrationTest.java]: This class is very very big, while it also contains tests for the split-up controllers. I would split the tests according to the changes made in #78

**Comment by TimvdLippe** [comment]
We might want to consider using Lombok for the DTO's. Other than that 1 comment regarding the `UserRestIntegrationTest`, then LGTM :+1:

File: src/main/java/ch/wisv/areafiftylan/dto/ProfileDTO.java (hunk 1 above)
Comment (skamoen) [inline src/main/java/ch/wisv/areafiftylan/dto/ProfileDTO.java]: See #95

File: src/test/java/ch/wisv/areafiftylan/UserRestIntegrationTest.java (hunk 2 above)
Comment (skamoen) [inline src/test/java/ch/wisv/areafiftylan/UserRestIntegrationTest.java]: It's a test class, testing all aspect of Users. What would we gain by splitting it up except for a couple of seconds when scrolling through? I don't think it's worth the extra files and lines in your debugger when running them.

**Pull request merged**